rwasm/runtime/wasmtime/host_function.go

```diff
@@ -0,0 +1,64 @@
+package runtimewasmtime
+
+import (
+	"fmt"
+
+	"github.com/bytecodealliance/wasmtime-go"
+	"github.com/pkg/errors"
+	"github.com/suborbital/reactr/rwasm/runtime"
+)
+
+var i32Type = wasmtime.NewValType(wasmtime.KindI32)
+
+// addHostFns adds a list of host functions to an import object
+func addHostFns(linker *wasmtime.Linker, fns ...runtime.HostFn) {
+	for i := range fns {
+		// we create a copy inside the loop otherwise things get overwritten
+		fn := fns[i]
+
+		params := make([]*wasmtime.ValType, fn.ArgCount)
+		for i := 0; i < fn.ArgCount; i++ {
+			params[i] = i32Type
```

Comment:
Maybe it's obvious, though I think a comment that explains that all host functions only operate on i32s would be good here.